<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(150, 150, 150);</string>
  </property>
  <widget class="QLabel" name="totalTimeTitle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-size: 28px;
color: white;
font-weight: 600;</string>
   </property>
   <property name="text">
    <string>Total Time Travelled</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalDistanceTitle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-size: 28px;
color: white;
font-weight: 600;</string>
   </property>
   <property name="text">
    <string>Total Distance Travelled</string>
   </property>
  </widget>
  <widget class="QLabel" name="numTripsTitle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-size: 28px;
color: white;
font-weight: 600;</string>
   </property>
   <property name="text">
    <string>Number of trips taken</string>
   </property>
  </widget>
  <widget class="QLabel" name="frequentStopOnTitle">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>240</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-size: 28px;
color: white;
font-weight: 600;
</string>
   </property>
   <property name="text">
    <string>Stop most frequently stepped on at </string>
   </property>
  </widget>
  <widget class="QLabel" name="frequentStopOffTitle">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>390</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-size: 28px;
color: white;
font-weight: 600;</string>
   </property>
   <property name="text">
    <string>Stop least frequently stepped off at </string>
   </property>
  </widget>
  <widget class="QLabel" name="streakTitle">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>550</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-size: 28px;
color: white;
font-weight: 600;
</string>
   </property>
   <property name="text">
    <string>Consequtive days travelled</string>
   </property>
  </widget>
  <widget class="QLabel" name="freqStopOn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>280</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-size: 48px;
color: white;
font-weight: 800;</string>
   </property>
   <property name="text">
    <string>Stop name </string>
   </property>
  </widget>
  <widget class="QLabel" name="freqStopOff">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>420</y>
     <width>501</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-size: 48px;
color: white;
font-weight: 800;</string>
   </property>
   <property name="text">
    <string>Stop name</string>
   </property>
  </widget>
  <widget class="QLabel" name="streak">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>580</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-size: 48px;
color: white;
font-weight: 800;
</string>
   </property>
   <property name="text">
    <string>XX Days</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeTravelled">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-size: 48px;
color: white;
font-weight: 800;</string>
   </property>
   <property name="text">
    <string>XX minutes</string>
   </property>
  </widget>
  <widget class="QLabel" name="distanceTravelled">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-size: 48px;
color: white;
font-weight: 800;</string>
   </property>
   <property name="text">
    <string>XX kilometers</string>
   </property>
  </widget>
  <widget class="QLabel" name="numTrips">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>590</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-size: 48px;
color: white;
font-weight: 800;</string>
   </property>
   <property name="text">
    <string>XX Trips</string>
   </property>
  </widget>
  <widget class="QLabel" name="freqStopOn_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>571</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: tahoma;
font-size: 60px;
color: white;
font-weight: 1200;</string>
   </property>
   <property name="text">
    <string>Personal Statistics</string>
   </property>
  </widget>
  <widget class="QLabel" name="frequentStopOnTitle_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font-family: tahoma;
font-weight:bold;
color: white;</string>
   </property>
   <property name="text">
    <string>View some interesting personal statistics from the last 7 days</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
